webpack.config.lib.js

```diff
@@ -36,18 +39,26 @@ module.exports = (env, argv) => ({
     ],
   },
   output: {
-    filename: '[name].js',
+    filename: argv.mode === 'production'
+      ? '[name].js'
+      : '[name].development.js',
     libraryTarget: 'umd',
     path: path.join(__dirname, 'dist'),
     publicPath: '/dist/',
   },
+  performance: {
+    maxAssetSize: argv.mode === 'production' ? 175000 : 1250000,
```

Comment:
Maybe we could hold these values in constants at the beginning of the file? I think it might be more handy when needed — we suppose to update the values from time to time.

Reply:
Yes, sure! I'm going to change it also in `webpack.config.demo.js` which is the source of this change.